fastify.js

```diff
@@ -78,6 +78,10 @@ function build (options) {
     started = true
   })
 
+  function throwIfAlreadyBinded (msg) {
```

Comment:
=> `throwIfAlreadyBound`

Reply:
thanks man, this is why I need always a reviewer!
Done